SU-HSDO/suhumsci
# SHS-5626: Quick fix on new accordion styles

# READY FOR REVIEW

## Summary
This ticket fixes the border-left color issue when accordion is opened on Colorful sites
<img width="1257" alt="Screenshot 2024-05-21 at 11 59 17 AM" src="https://github.com/SU-HSDO/suhumsci/assets/48533432/39f78545-18c4-47d6-8a32-c36db8c47ea2">


## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. Create a Flexible Page on Colorful
2. Add an Accordion component and select the "Light" variation on styles
3. Confirm the border-left color when accordion is opened, it's the same (like the image below)

<img width="1269" alt="Screenshot 2024-05-21 at 12 33 39 PM" src="https://github.com/SU-HSDO/suhumsci/assets/48533432/d95ad2c9-845c-4adc-a0c5-ae049f20ea4f">



## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
